tests/externs.el

```diff
@@ -59,3 +59,47 @@
 ;;;TODO
 ;; ensure that any symbols bound with the import syntax are added to the extern list
 ;; ensure that any symbols bound with the export syntax exist in the file scope
+
+;; Test `/*jslint*/` declarations
+
+(ert-deftest js2-finds-jslint-declaration-externs ()
+  (with-temp-buffer
+    (insert "/*jslint browser, node:false, es6: true */")
```

Comment:
Yes, `node: true` should work and I have definitely seen this in the code that initially made me want this feature in `js2-mode`. :D